controllers/clusterurlmonitor/clusterurlmonitor_supplement.go

```diff
@@ -110,6 +110,8 @@ func (s *ClusterUrlMonitorReconciler) EnsureMonitorAndDependenciesAbsent(cluster
 	}
 
 	if s.Common.DeleteFinalizer(&clusterUrlMonitor, FinalizerKey) {
+		// ignore the output as we want to remove the PrevFinalizerKey anyways
+		s.Common.DeleteFinalizer(&clusterUrlMonitor, PrevFinalizerKey)
```

Comment:
What if we fail to delete the prevfinalizer?

Reply:
same